### DISCUSSION TITLE
Is there any plan to support HTTP/3? or anything relavated?

### DISCUSSION THREAD

**Discussion by huangjj27** [rwf2/Rocket#2345] [Questions] [answered]
I'm a fan of QUIC and HTTP/3, and I want to know if rocket will support HTTP/3 ?

**Comment by the10thWiz** [comment, marked as answer]
Rocket currently relies on `hyper` for it's HTTP backend, and hyper currently only supports HTTP 1 & 2. HTTP/3 support would be nice, but it would either require hyper to support it (which is not likely to be soon), or Rocket to switch to alternative backend that does support it. Sergio does have some interest in moving away from hyper, but this is unlikely to happen anytime soon.
